docs/semgrep-sc/sc-overview.md

```diff
@@ -111,11 +111,13 @@ The following table displays differences between Semgrep and Semgrep Supply Chai
 
 Refer to [Supported languages](/docs/supported-languages#semgrep-supply-chain) to see all languages supported by Semgrep Supply Chain.
 
-## Limitations of Semgrep Supply Chain scans
+## Transitive dependencies and reachability analysis
 
-* Semgrep Supply Chain supports scanning for transitive or indirect dependencies for all of its supported languages **except Java**. Findings are collected and displayed in **Semgrep App > Supply Chain**.
-* Semgrep Supply Chain detects transitivity only for **Python and JavaScript **languages.
-* Semgrep Supply Chain generates reachable findings only for direct dependencies.
+See [SSC glossary > Transitivity](/docs/semgrep-sc/sc-glossary/#transitive-or-indirect-dependency) for a definition of a transitive dependency.
+
+* Semgrep Supply Chain does **not** perform reachability analysis for transitive dependencies.
```

Comment:
I would add an extra sentence to clarify here like "Meaning we do not scan the source code of your dependencies to determine if any of _their_ dependencies might produce a reachable finding in your code"

Reply:
Added that bit in! Thanks :)